crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -62,8 +62,8 @@ def transition(inst, name, state)
         return [404, "Node not found"]
       end
 
-      if state == "readying"
-        transition_to_readying inst, name, state, node
+      if state == "ready" && !node.admin?
```

Comment:
I think you're right for Cloud 5; take my comment about moving to the cookbook as input when forward-porting the fix ;-)